### DISCUSSION TITLE
Scrum Prep Milestone 2 - Week 5

### DISCUSSION THREAD

**Discussion by nelson-escalona** [uprm-inso4101-2024-2025-s1/semester-project-regiupr#139] [Scrum Preps]
Scrum Prep Milestone 2 - Week 5

**Comment by christianmedina-dev** [comment]
Login Team. Focused on completing Lecture Tasks #130 and #140 after finishing with Sign Up Form Issue #86.

**Comment by keisanabria** [comment]
Documented the minute of the scrum meeting of Friday, 18 October 2024, and completed the task of taking minute for week 3 (which corresponded to that minute). Additionally, I will be documenting tomorrow, Friday, 25 October's minute to complete the task regarding this. Other than this, I was issued the task to add the diagrams that showed a visual of how algorithms worked in College Planning, Sequence Diagram for Profile Screen feature, and others. The latter was completed and was added to the Documentation (section 2.3.1). On the other hand, I am working on creating two new Issues and completing them before the end of Milestone 2.

**Comment by carlospepin23** [comment]
Completed Task **#96** and Lecture Topic Task **#123** with @nelson-escalona .

**Comment by J3SSY-ANDU** [comment]
Completed tasks #114, #115, #116 and #117 with @PedroRodz

**Comment by Gian-Go** [comment]
Worked on updating the logbook to coincide with relevant completed tasks within milestone 2. Also worked on improving weaker aspects of the documentation to ensure it complies with guidelines.

**Comment by Yamil-Serrano** [comment]
Completed tasks #121, #89 and working on task #120 (a lecture topic task) with @cristianMarcial.

**Comment by PedroRodz** [comment]
Closed some tasks. Gave some feedback on documentation.

**Comment by Carloscaban** [comment]
Having already completed tasks, this week I couldn't do anything, I had a hellweek.

**Comment by cristianMarcial** [comment]
> Completed tasks #121, #89 and working on task #120 (a lecture topic task) with @cristianMarcial.

👆
